### ISSUE TITLE
Created enclave has an invalid metadata magic

### ISSUE THREAD

**Issue by achow101** [LedgerHQ/bolos-enclave#3]
I'm trying to create an enclave that can be used in SGX simulated mode but when I do, the resulting enclave gives me error code 8201 which is SGX_ERROR_INVALID_METADATA according to Intel's sgx documentation. After some further debugging, I have come to the conclusion that the enclave's metadata magic bytes are incorrect.

How can this be fixed? Or is there some other way to run the bolos enclave in a simulator without needing hardware that supports Intel SGX?

**Comment by btchip** [comment]
yes, you can build using Makefile.simu both in the tee/ and host/ directories then use testTee to run a test program that doesn't require SGX. Note that this project is fairly outdated now and will be updated in the coming months.
